src/App.js

```diff
@@ -1,29 +1,60 @@
 import React, { useEffect, useState } from "react";
 import fetchCountries from "./api/fetchCountries";
+import fetchCityImage from "./api/fetchCityImage";
+import fetchWeather from "./api/fetchWeather";
+import { GridLoader } from "react-spinners";
 
 function App(response) {
+  const [loading, setLoading] = useState(false);
+
   const [countries, setCountries] = useState();
   const [country, setCountry] = useState("United Kingdom");
-  const [capital, setcapital] = useState("London");
+  const [capital, setCapital] = useState("London");
+  const [cityImage, setCityImage] = useState("London");
+  const [weather, setWeather] = useState("raining");
```

Comment:
`weather` is initialized as a string, but then is fetched as a nested object. Should this be initialized to undefined/null?

Reply:
You can merge after fixing this